src/chrome/content/rules/Aeriagames.xml

```diff
@@ -10,28 +17,25 @@
 			- s.aeriastatic.com
 
 
-	Problematic domains:
+		Problematic domains:
 
-		- s.aeriastatic.com	(works, akamai)
+			- s.aeriastatic.com	(works, akamai)
 
--->
-<ruleset name="Aeriagames">
-<target host="www.aeriagames.com" />
-<target host="aeriagames.com" />
-	<target host="*.aeriastatic.com" />
 
-<rule from="^http://(?:www\.)?aeriagames\.com/" to="https://www.aeriagames.com/"/>
-
-	<rule from="^http://c\.aeriastatic\.com/"
-		to="https://c.aeriastatic.com/" />
-
-	<!--	We must rewrite from https://s to c due to protocol-relative
+		We must rewrite from https://s to c due to protocol-relative
```

Comment:
But we don't as far as I can tell (?). So either this comment is outdated and can be removed or an additional rule should be added to this ruleset.